### PR TITLE
Create visa.md

### PR DESCRIPTION
**Pull request by hellvix**
Created page with instructions on how to request a tourist visa for the schengen area.

### CODE REVIEW

File: hugo_site/content/visa.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# VISA
+If your country is not part of the [Schengen Agreement](https://www.schengenvisainfo.com/who-needs-schengen-visa/), you will need to request a tourist VISA in the Danish Embassy. We have spoken to the Danish Immigration Office and below we have compiled a list of documents you will need to provide when requesting the travel document. Have in mind that this process can take some months, therefore we recommend starting it as soon as possible.
```
[the remark this answers is not in the document]
Comment (hellvix) [reply hugo_site/content/visa.md]: Every embassy/consulate is just a proxy to Udlændingestyrelsen. The latter does all the processing of visas and permits. In their website they mention it is up to 90 days: https://www.nyidanmark.dk/da/Applying/Short%20stay%20visa/Private%20and%20turist%20visa

File: hugo_site/content/visa.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Visa
-If your country is not part of the [Schengen Agreement](https://www.schengenvisainfo.com/who-needs-schengen-visa/), you will need to request a tourist Visa in the Danish Embassy. We have spoken to the Danish Immigration Office and below we have compiled a list of attachments you will need to provide when requesting the travel document. Have in mind that this process can take up to 90 days, therefore we recommend starting it as soon as possible.
+If your country is not part of the [Schengen Agreement](https://www.schengenvisainfo.com/who-needs-schengen-visa/), you will need to request a tourist Visa in the Danish Embassy. We have spoken to the Danish Immigration Office and below we have compiled a list of attachments you will need to provide when requesting the travel document. Contact the local embassy for information on processing time.
```
Comment (benjaoming) [inline hugo_site/content/visa.md]: Was this part incorrect since you removed it?

> Have in mind that this process can take up to 90 days

**Comment by benjaoming** [comment]
Great work @hellvix !

File: hugo_site/content/visa.md (hunk 2 above)
Comment (hellvix) [reply hugo_site/content/visa.md]: I found out the processing time depends on each embassy. They are the ones processing these visas. Only upper level requests are dealt by Udlændingestyrelsen, such as work and residence permits. So lets leave it unmentioned.

Comment (benjaoming) [reply hugo_site/content/visa.md]: That's cool - yeah probably people know their own countries issues and timelines better than we can tell or hint.

**Pull request merged**